security.rst

```diff
@@ -342,14 +342,12 @@ Guard Authenticators
 ....................
```

Comment:
```suggestion
~~~~~~~~~~~~~~~~~~~~
```
Because you changed the title level of item 3b the level of Guard Authenticators needs to be changed too.

Reply:
Thanks - looks like I finally got it now :-)